test/framework/management/kind/mgmt.go

```diff
@@ -0,0 +1,239 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kind
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/cluster-api/test/framework/exec"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Shells out to `kind`, `kubectl`
+
+// Cluster represents a Kubernetes cluster used as a management cluster backed by kind.
+type Cluster struct {
+	Name              string
+	KubeconfigPath    string
+	Client            client.Client
+	Scheme            *runtime.Scheme
+	WorkerKubeconfigs map[string]string
```

Comment:
This is storing kubeconfigs for workload clusters?

Reply:
Can we rename this to be clearer, maybe `WorkloadClusterKubeconfigs` or `ClusterKubeconfigs` ?

Reply:
Renamed